src/index.js

```diff
@@ -55,7 +55,7 @@ if (!owner || !repo) {
   try {
     const gitconfig = readFileSync(path.join(dir, '.git/config'), 'utf-8');
     const remoteOrigin = ini.parse(gitconfig)['remote "origin"'];
-    const match = remoteOrigin.url.match(/.+:([^/]+)\/(.+)\.git/);
+    const match = remoteOrigin.url.match(/[:/]([^/]+?)\/([^/]+?)(?:\.git)?$/);
```

Comment:
Since this project is aimed specifically at GitHub repos, it might be worth making this explicit in the regex:

```suggestion
    const match = remoteOrigin.url.match(/github.com[:/]([^/]+?)\/([^/]+?)(?:\.git)?$/);
```

...which also makes the pattern more readable/understandable IMO.

Reply:
Also, the non-greedy quantifier `+?` is redundant since the `[^/]` pattern is followed immediately by `/`, which makes it necessarily extend all the way to the first `/` (and never past it, since it's by definition the negation of `/`).

I would suggest removing the two `?`s, as they are serving no purpose here, and only add to the visual noise that regex already naturally has :)

Reply:
The last non-greedy can't be removed, otherwise the:

<img width="832" alt="Screenshot 2022-06-12 at 11 21 15" src="https://user-images.githubusercontent.com/1017236/173228732-f4eb1051-a8d1-4b00-8b8c-8bd9f8b6aa21.png">
 
 The first one is unnecessary however.

Reply:
Added `github.com` as suggested.

Reply:
Also changed the repo capture group back to `.+?` as it was an unnecessary change.

Reply:
> The last non-greedy can't be removed

Oh, right, in that case there's no `/` at the end — sorry for the overlook :)

Reply:
> Also changed the repo capture group back to `.+?` as it was an unnecessary change.

Yeah, good call. The characters acceptable in a GitHub repo slug are much more restricted than either `.` or `[^/]`, so one might as well use the easier-to-read variant since neither provides an accuracy benefit.

Reply:
@waldyrious no worries and thanks for your review ❤️

Reply:
My pleasure! I'm just glad there are still people at Uphold caring for the open source packages :D